### PR TITLE
Adds common-sign-language-0001 model

### PR DESCRIPTION
**Pull request by evgeny-izutov**

### CODE REVIEW

File: models/public/common-sign-language-0001/common-sign-language-0001.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Batch of images of the shape [1x3x8x224x224] in the [BxCxTxHxW] format, where:
+
+- `B` - batch size
+- `C` - channel
+- `T` - sequence length
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
```
Comment (eaidova) [inline models/public/common-sign-language-0001/common-sign-language-0001.md:48]: is it impossible to convert network with bgr channels order?
Can we use --reverse_input_channels option during MO conversion?

Comment (evgeny-izutov) [reply models/public/common-sign-language-0001/common-sign-language-0001.md:48]: Unfortunately MO supports `--reverse_input_channels` parameter for 4-dimensional input only but the network input is 5-dimensional. So, the answer is no.

**Review by eaidova — APPROVED** [review]

File: models/public/common-sign-language-0001/common-sign-language-0001.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Specification
+
+| Metric                                  | Value        |
+|-----------------------------------------|--------------|
+| Top-1 accuracy (continuous Jester\*)    | 93.58%       |
+| GFlops                                  | 4.2269       |
+| MParams                                 | 4.1128       |
+| Source framework                        | PyTorch\*    |
+
+## Performance
```
Comment (vladimir-dudnik) [inline models/public/common-sign-language-0001/common-sign-language-0001.md:22]: we are removing empty sections from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please adopt your description accordingly

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

File: models/public/common-sign-language-0001/common-sign-language-0001.md (hunk 2 above)
Comment (evgeny-izutov) [reply models/public/common-sign-language-0001/common-sign-language-0001.md]: Done.

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**